### PR TITLE
[14][FWD] [FIX] Disable unarchiving buffer on product activation

### PR DESCRIPTION
**Pull request by TDu**
Forward port of https://github.com/OCA/ddmrp/pull/180

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: ddmrp/models/product_product.py [hunk 1]
```diff
@@ -15,14 +15,7 @@ class Product(models.Model):
...
-                [
-                    ("product_id", "in", self.ids),
-                    "|",
-                    ("active", "=", True),
-                    ("active", "=", False),
-                ]
-            )
-            buffers.write({"active": values.get("active")})
+        if values.get("active") is False:
+            buffers = self.env["stock.buffer"].search([("product_id", "in", self.ids)])
```
Comment (simahawk) [inline ddmrp/models/product_product.py]:
```suggestion
            buffers = self.env["stock.buffer"].sudo().search([("product_id", "in", self.ids)])
```

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by LoisRForgeFlow** [comment]
@TDu thanks, can you remove the last 2 commits and keep only the 1st one?

**Comment by TDu** [comment]
@LoisRForgeFlow  Done

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**